source/install.txt

```diff
@@ -1,177 +1,190 @@
-===============
-Install Compass
-===============
+============================
+Download and Install Compass
+============================
 
 .. default-domain:: mongodb
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
+.. include:: /includes/styles/corrections.rst
 
-Compatibility
--------------
+.. tabs-pillstrip:: platforms
 
-|compass| works with MongoDB Server version 3.0 and higher, both
-Enterprise and Community editions. It runs on 64-bit versions
-of Mac OS X, Windows, Ubuntu 14.04+ and RHEL 7+.
+Software Requirements
+---------------------
 
-Software requirements to run |compass-short| are listed below:
+|compass-short| requires:
 
-.. include:: /includes/fact-compass-platforms.rst
+.. tabs-platforms::
 
-Installation
-------------
+   tabs:
+     - id: macos
+       content: |
 
-.. note::
+         - 64-bit version of macOS 10.10 or later
+         - MongoDB 3.0 or later.
 
-   As of MongoDB Server version 3.6, MongoDB packages a
-   platform-specific |compass-short| installation script with
-   `MongoDB Server downloads
-   <https://www.mongodb.com/download-center/enterprise>`_. For details,
-   see the :manual:`install_compass script
-   </reference/program/install_compass/>` documentation.
+     - id: windows
+       content: |
 
-.. tabs::
+         - 64-bit version of Microsoft Windows 7 or later
```

Comment:
Add period at the end, since other bullet(s) has / have period(s) at the end.

Reply:
Agreed. Fixed.